docs/src/tutorials/tutorial_1.md

```diff
@@ -2,6 +2,350 @@
+If you are reading this tutorial, I am going to assume this is the first time you are using `Javis` to create an animation. 
+In that case, welcome to `Javis`! 😃 
+By following this tutorial, we are going to make you a director of your very own animations written in pure Julia! 🎬 🎥
 
 If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](../index.html#Installation).
+
+## Learning Outcomes
+
+In this tutorial you'll learn:
+
+- What `Javis.jl` is.
+- How to make a basic animation using `Action` objects.
+- Move objects created by `Javis`.
+- Elements of `Luxor.jl`.
+
+By the end of this tutorial, you will have made the following animation:
+
+![](assets/dancing_circles.gif)
+
+With all that said, let's dive in to this tutorial! ✨
+
+## So... What Is `Javis.jl`?
+
+`Javis.jl` is an abstraction on top of powerful graphics tools to make animations and visualizations easy to make.
+It is built on top of the fantastic Julia animation packages, [`Luxor.jl`](https://github.com/JuliaGraphics/Luxor.jl) and [`Cairo.jl`](https://github.com/JuliaGraphics/Cairo.jl).
+`Cairo.jl` is much too complex to explain here, but `Luxor.jl` gives one the ability to define and draw on a canvas.
+`Luxor.jl` provides simple functions like `line`, `circle` and `Draw` by which one can make animations. 
+
+> **NOTE:** If you're interested in 2D graphics, you should definitely check out the awesome `Luxor.jl` package.
+> It has a [great tutorial](https://juliagraphics.github.io/Luxor.jl/stable/tutorial/) that will give you an even greater understanding of how `Javis.jl` works.
+
+## The `javis` Function - the Heart of `Javis.jl`
+
+In order to use `Javis`, we will start with the following import:
+
+```julia
+using Javis
+
+```
+
+Then, we will introduce our first and most import function from `Javis` - the `javis` function:
+
+```
+javis(
+    video = [[video]], 
+    actions = [[actions]], 
+    pathname = [[pathname]]
+)
+```
+
+The `javis` function has three key word parameters that the user must define in order to create an animation:
+
+- `video` - a `Video` struct
+- `actions` - a vector of `Action` objects
+- `pathname` - a `String` 
+
+These kwargs are further explained in the following sections. 
+
+### `video`
+
+The `video` argument requires a `Video` struct.
+This `Video` struct creates the canvas upon which we will draw. 🎨 
+To create a `Video` object, one defines the constructor with a width and height:
+
+```julia
+myvideo = Video(500,500) # 500 x 500 // width x height
+
+```
+
+### `actions`
+
+Core to not only the `javis` function but the entire `Javis.jl` library are `Action` objects. 
+`Action` objects will be better explained in [Tutorial 2](tutorial_2.html).
+For now, just know that an `Action` object is what draws shapes or moves shapes on a `Video`. In the `javis` function, the `actions` kwarg is a vector of `Action` objects.
```

Comment:
it is not actually a `kwarg` but I know you love `kwarg` :heart:

Reply:
![image](https://user-images.githubusercontent.com/29561456/90322961-f1678280-df28-11ea-8c4b-2845604cd764.png)